### PR TITLE
feature: add electoral districts to belgium

### PR DESCRIPTION
**Pull request by zikowang**
Added provinces to Belgium file.

Added Readme file with source for Belgian provinces.

### CODE REVIEW

**Comment by zikowang** [comment]
@jloutsenhizer @evannjw @jpmckinney 

Have a quick look, please.

File: identifiers/country-be/provinces.csv [hunk 1]
```diff
@@ -0,0 +1,12 @@
+id,name
+ocd-division/country:be/province:an,Antwerp
+ocd-division/country:be/province:be,Brussels-Capital
```
Comment (jpmckinney) [inline identifiers/country-be/provinces.csv:3]: Brussels Capital is not a province according to https://en.wikipedia.org/wiki/ISO_3166-2:BE

File: identifiers/country-be/provinces.csv [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+ocd-division/country:be/province:vb,Flemish-Brabant
+ocd-division/country:be/province:bw,Walloon-Brabant
+ocd-division/country:be/province:wv,West-Flanders
+ocd-division/country:be/province:ov,East-Flanders
+ocd-division/country:be/province:ht,Hainaut
+ocd-division/country:be/province:lg,Liège
+ocd-division/country:be/province:li,Limburg
+ocd-division/country:be/province:na,Namur
+ocd-division/country:be/province:lx,Luxembourg
```
[the remark this answers is not in the document]
Comment (zikowang) [reply identifiers/country-be/provinces.csv]: The 2 letters are HASC and come from [Wiki Provinces_of_Belgium](https://en.wikipedia.org/wiki/Provinces_of_Belgium).

I changed them using ISO 3166-2 now.

**Review by evannjw — APPROVED** [review]

File: identifiers/country-be.csv [hunk 3]
```diff
@@ -1,2 +1,13 @@
...
+ocd-division/country:be/ed:brussels-capital,Brussels-Capital
+ocd-division/country:be/ed:east-flanders,East-Flanders
+ocd-division/country:be/ed:flemish-brabant,Flemish-Brabant
+ocd-division/country:be/ed:hainaut,Hainaut
+ocd-division/country:be/ed:liege,Liège
+ocd-division/country:be/ed:limburg,Limburg
+ocd-division/country:be/ed:luxembourg,Luxembourg
+ocd-division/country:be/ed:namur,Namur
+ocd-division/country:be/ed:walloon-brabant,Walloon-Brabant
+ocd-division/country:be/ed:west-flanders,West-Flanders
```
Comment (jpmckinney) [inline identifiers/country-be.csv:13]: Correspond to Wikipedia

```suggestion
ocd-division/country:be/ed:antwerp,Antwerp
ocd-division/country:be/ed:brussels,Brussels
ocd-division/country:be/ed:east-flanders,East Flanders
ocd-division/country:be/ed:flemish-brabant,Flemish Brabant
ocd-division/country:be/ed:hainaut,Hainaut
ocd-division/country:be/ed:liege,Liège
ocd-division/country:be/ed:limburg,Limburg
ocd-division/country:be/ed:luxembourg,Luxembourg
ocd-division/country:be/ed:namur,Namur
ocd-division/country:be/ed:walloon-brabant,Walloon Brabant
ocd-division/country:be/ed:west-flanders,West Flanders
```

**Comment by jpmckinney** [comment]
(I made the suggested edit to the wrong file.)

File: identifiers/country-be.csv (hunk 3 above)
Comment (zikowang) [reply identifiers/country-be.csv]: Thank you, looks legit.

**Comment by zikowang** [comment]
Electoral district names updated.

**Review by jpmckinney — APPROVED** [review]

**Review by jloutsenhizer — APPROVED** [review]

**Pull request merged**